modules/simple_forms_api/app/services/simple_forms_api/notification/email.rb

```diff
@@ -33,12 +33,20 @@ def send(at: nil)
         template_id = TEMPLATE_IDS[form_number][notification_type.to_s]
         return unless template_id
 
-        sent_to_va_notify = if at
-                              enqueue_email(at, template_id)
-                            else
-                              send_email_now(template_id)
-                            end
-        StatsD.increment('silent_failure', tags: statsd_tags) if error_notification? && !sent_to_va_notify
+        email_job_id = if at
+                         enqueue_email(at, template_id)
+                       else
+                         send_email_now(template_id)
+                       end
+
+        if email_job_id
+          Rails.logger.info('Simple Forms - Email job enqueued', email_job_id:, confirmation_number:)
+        elsif error_notification?
+          StatsD.increment('silent_failure', tags: statsd_tags)
+          Rails.logger.error('Simple Forms - Error email job failed to enqueue', confirmation_number:)
+        else
+          Rails.logger.error('Simple Forms - Non-error email job failed to enqueue', confirmation_number:)
+        end
```

Comment:
Minor suggestion is to use a guard clause to avoid the multiple branch if statement:

```suggestion
        Rails.logger.info('Simple Forms - Email job enqueued', email_job_id:, confirmation_number:) if email_job_id
        
        if error_notification?
          StatsD.increment('silent_failure', tags: statsd_tags)
          Rails.logger.error('Simple Forms - Error email job failed to enqueue', confirmation_number:)
        else
          Rails.logger.error('Simple Forms - Non-error email job failed to enqueue', confirmation_number:)
        end
```

Reply:
This is different behavior. If it's an `error_notification` and there is an `email_job_id` we don't want to increment statsd, we still want the success log. The alternative might be a nested conditional but I think this is preferable.

Reply:
Oops, my mistake. I thought error_notification? only returned if there was an error with the notification 😅